### DISCUSSION TITLE
Why is there no response when you enter% time%?

### DISCUSSION THREAD

**Discussion by QianMuu** [PlaceholderAPI/PlaceholderAPI#665] [(2) Support] [answered]
Why is there no response when you enter% time%?

**Comment by Andre601** [comment, marked as answer]
This is not PAPI related as the Placeholder format is `%placeholder_text%` and not `text%`

**Reply by QianMuu** [reply]
So, how do I enter the time?

**Reply by Andre601** [reply]
We don't know what you're trying here

**Comment by RoinujNosde** [comment]
https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#localtime
